--- proxmox-backup/Cargo.toml	2023-02-22 11:22:52.178824909 +0100
+++ proxmox-backup/Cargo.gem.toml	2023-02-22 11:13:23.082613194 +0100
@@ -43,7 +43,6 @@
     "proxmox-backup-banner",
     "proxmox-backup-client",
     "proxmox-file-restore",
-    "proxmox-restore-daemon",
     "proxmox-rrd",
 
     "pxar-bin",
@@ -62,9 +61,6 @@
 proxmox-http = { version = "0.7", features = [ "client", "http-helpers", "websocket" ] } # see below
 proxmox-io = "1.0.1" # tools and client use "tokio" feature
 proxmox-lang = "1.1"
-proxmox-ldap = "0.1"
-proxmox-metrics = "0.2"
-proxmox-rest-server = "0.2.2"
 # some use "cli", some use "cli" and "server", pbs-config uses nothing
 proxmox-router = { version = "1.3.1", default_features = false }
 # everything but pbs-config and pbs-client ues "api-macro"
@@ -72,7 +68,6 @@
 proxmox-section-config = "1"
 proxmox-serde = "0.1.1"
 proxmox-shared-memory = "0.2.3"
-proxmox-subscription = { version = "0.3", features = [ "api-types" ] }
 proxmox-sys = "0.4.2"
 proxmox-tfa = { version = "2.1", features = [ "api", "api-types" ] }
 proxmox-time = "1.1.2"
@@ -80,9 +75,6 @@
 
 # other proxmox crates
 pathpatterns = "0.1.2"
-proxmox-acme-rs = "0.4"
-proxmox-apt = "0.9.0"
-proxmox-openid = "0.9.9"
 pxar = "0.10.2"
 
 # PBS workspace
@@ -101,7 +93,6 @@
 
 # regular crates
 anyhow = "1.0"
-apt-pkg-native = "0.3.2"
 base64 = "0.13"
 bitflags = "1.2.1"
 bytes = "1.0"
@@ -154,7 +145,6 @@
 
 [dependencies]
 anyhow.workspace = true
-apt-pkg-native.workspace = true
 base64.workspace = true
 bitflags.workspace = true
 bytes.workspace = true
@@ -206,24 +196,17 @@
 proxmox-http = { workspace = true, features = [ "client-trait", "proxmox-async" ] } # pbs-client doesn't use these
 proxmox-io.workspace = true
 proxmox-lang.workspace = true
-proxmox-ldap.workspace = true
-proxmox-metrics.workspace = true
-proxmox-rest-server.workspace = true
 proxmox-router = { workspace = true, features = [ "cli", "server"] }
 proxmox-schema = { workspace = true, features = [ "api-macro" ] }
 proxmox-section-config.workspace = true
 proxmox-serde = { workspace = true, features = [ "serde_json" ] }
 proxmox-shared-memory.workspace = true
-proxmox-subscription.workspace = true
 proxmox-sys = { workspace = true, features = [ "sortable-macro", "timer" ] }
 proxmox-tfa.workspace = true
 proxmox-time.workspace = true
 proxmox-uuid.workspace = true
 
 pathpatterns.workspace = true
-proxmox-acme-rs.workspace = true
-proxmox-apt.workspace = true
-proxmox-openid.workspace = true
 pxar.workspace = true
 
 pbs-api-types.workspace = true
@@ -240,28 +223,30 @@
 # Local path overrides
 # NOTE: You must run `cargo update` after changing this for it to take effect!
 [patch.crates-io]
-#proxmox-acme-rs = { path = "../proxmox-acme-rs" }
-#proxmox-apt = { path = "../proxmox-apt" }
-#proxmox-async = { path = "../proxmox/proxmox-async" }
-#proxmox-compression = { path = "../proxmox/proxmox-compression" }
-#proxmox-borrow = { path = "../proxmox/proxmox-borrow" }
-#proxmox-fuse = { path = "../proxmox-fuse" }
-#proxmox-http = { path = "../proxmox/proxmox-http" }
-#proxmox-io = { path = "../proxmox/proxmox-io" }
-#proxmox-lang = { path = "../proxmox/proxmox-lang" }
-#proxmox-openid = { path = "../proxmox-openid-rs" }
-#proxmox-router = { path = "../proxmox/proxmox-rest-server" }
-#proxmox-router = { path = "../proxmox/proxmox-router" }
-#proxmox-schema = { path = "../proxmox/proxmox-schema" }
-#proxmox-section-config = { path = "../proxmox/proxmox-section-config" }
-#proxmox-shared-memory = { path = "../proxmox/proxmox-shared-memory" }
-#proxmox-sys = { path = "../proxmox/proxmox-sys" }
-#proxmox-serde = { path = "../proxmox/proxmox-serde" }
-#proxmox-tfa = { path = "../proxmox/proxmox-tfa" }
-#proxmox-time = { path = "../proxmox/proxmox-time" }
-#proxmox-uuid = { path = "../proxmox/proxmox-uuid" }
-#pxar = { path = "../pxar" }
+proxmox-async = { path = "../proxmox/proxmox-async" }
+proxmox-compression = { path = "../proxmox/proxmox-compression" }
+proxmox-borrow = { path = "../proxmox/proxmox-borrow" }
+proxmox-fuse = { path = "../proxmox-fuse" }
+proxmox-http = { path = "../proxmox/proxmox-http" }
+proxmox-io = { path = "../proxmox/proxmox-io" }
+proxmox-lang = { path = "../proxmox/proxmox-lang" }
+proxmox-router = { path = "../proxmox/proxmox-router" }
+proxmox-schema = { path = "../proxmox/proxmox-schema" }
+proxmox-section-config = { path = "../proxmox/proxmox-section-config" }
+proxmox-shared-memory = { path = "../proxmox/proxmox-shared-memory" }
+proxmox-sys = { path = "../proxmox/proxmox-sys" }
+proxmox-serde = { path = "../proxmox/proxmox-serde" }
+proxmox-tfa = { path = "../proxmox/proxmox-tfa" }
+proxmox-time = { path = "../proxmox/proxmox-time" }
+proxmox-uuid = { path = "../proxmox/proxmox-uuid" }
+pxar = { path = "../pxar" }
 
 [features]
 default = []
-#valgrind = ["valgrind_request"]
+
+[package.metadata.generate-rpm]
+assets = [
+  { source = "target/release/pxar", dest = "/usr/bin/pxar", mode = "0755" },
+  { source = "target/release/proxmox-backup-client", dest = "/usr/bin/proxmox-backup-client", mode = "0755" }
+]
+
