--- proxmox-backup/Cargo.toml	2023-03-30 20:44:58.107032607 +0200
+++ proxmox-backup/Cargo.gem.toml	2023-03-30 22:07:03.000000000 +0200
@@ -42,7 +42,7 @@
     "proxmox-backup-banner",
     "proxmox-backup-client",
     "proxmox-file-restore",
-    "proxmox-restore-daemon",
+    #"proxmox-restore-daemon",
     "proxmox-rrd",
 
     "pxar-bin",
@@ -62,9 +62,9 @@
 proxmox-http = { version = "0.8.0", features = [ "client", "http-helpers", "websocket" ] } # see below
 proxmox-io = "1.0.1" # tools and client use "tokio" feature
 proxmox-lang = "1.1"
-proxmox-ldap = "0.1"
-proxmox-metrics = "0.2"
-proxmox-rest-server = { version = "0.3.0", features = [ "templates" ] }
+#proxmox-ldap = "0.1"
+#proxmox-metrics = "0.2"
+#proxmox-rest-server = { version = "0.3.0", features = [ "templates" ] }
 # some use "cli", some use "cli" and "server", pbs-config uses nothing
 proxmox-router = { version = "1.3.1", default_features = false }
 # everything but pbs-config and pbs-client ues "api-macro"
@@ -73,7 +73,7 @@
 proxmox-serde = "0.1.1"
 proxmox-shared-memory = "0.2.3"
 proxmox-sortable-macro = "0.1.2"
-proxmox-subscription = { version = "0.3", features = [ "api-types" ] }
+#proxmox-subscription = { version = "0.3", features = [ "api-types" ] }
 proxmox-sys = "0.4.2"
 proxmox-tfa = { version = "3", features = [ "api", "api-types" ] }
 proxmox-time = "1.1.2"
@@ -81,9 +81,9 @@
 
 # other proxmox crates
 pathpatterns = "0.1.2"
-proxmox-acme-rs = "0.4"
-proxmox-apt = "0.9.0"
-proxmox-openid = "0.9.9"
+#proxmox-acme-rs = "0.4"
+#proxmox-apt = "0.9.0"
+#proxmox-openid = "0.9.9"
 pxar = "0.10.2"
 
 # PBS workspace
@@ -101,7 +101,7 @@
 
 # regular crates
 anyhow = "1.0"
-apt-pkg-native = "0.3.2"
+#apt-pkg-native = "0.3.2"
 base64 = "0.13"
 bitflags = "1.2.1"
 bytes = "1.0"
@@ -154,7 +154,7 @@
 
 [dependencies]
 anyhow.workspace = true
-apt-pkg-native.workspace = true
+#apt-pkg-native.workspace = true
 base64.workspace = true
 bitflags.workspace = true
 bytes.workspace = true
@@ -207,25 +207,25 @@
 proxmox-http = { workspace = true, features = [ "client-trait", "proxmox-async", "rate-limited-stream" ] } # pbs-client doesn't use these
 proxmox-io.workspace = true
 proxmox-lang.workspace = true
-proxmox-ldap.workspace = true
-proxmox-metrics.workspace = true
-proxmox-rest-server = { workspace = true, features = [ "rate-limited-stream" ] }
+#proxmox-ldap.workspace = true
+#proxmox-metrics.workspace = true
+#proxmox-rest-server = { workspace = true, features = [ "rate-limited-stream" ] }
 proxmox-router = { workspace = true, features = [ "cli", "server"] }
 proxmox-schema = { workspace = true, features = [ "api-macro" ] }
 proxmox-section-config.workspace = true
 proxmox-serde = { workspace = true, features = [ "serde_json" ] }
 proxmox-shared-memory.workspace = true
 proxmox-sortable-macro.workspace = true
-proxmox-subscription.workspace = true
+#proxmox-subscription.workspace = true
 proxmox-sys = { workspace = true, features = [ "timer" ] }
 proxmox-tfa.workspace = true
 proxmox-time.workspace = true
 proxmox-uuid.workspace = true
 
 pathpatterns.workspace = true
-proxmox-acme-rs.workspace = true
-proxmox-apt.workspace = true
-proxmox-openid.workspace = true
+#proxmox-acme-rs.workspace = true
+#proxmox-apt.workspace = true
+#proxmox-openid.workspace = true
 pxar.workspace = true
 
 pbs-api-types.workspace = true
@@ -242,31 +242,36 @@
 # NOTE: You must run `cargo update` after changing this for it to take effect!
 [patch.crates-io]
 #proxmox-acme-rs = { path = "../proxmox-acme-rs" }
-#proxmox-async = { path = "../proxmox/proxmox-async" }
-#proxmox-auth-api = { path = "../proxmox/proxmox-auth-api" }
-#proxmox-borrow = { path = "../proxmox/proxmox-borrow" }
-#proxmox-compression = { path = "../proxmox/proxmox-compression" }
-#proxmox-fuse = { path = "../proxmox-fuse" }
-#proxmox-http = { path = "../proxmox/proxmox-http" }
-#proxmox-io = { path = "../proxmox/proxmox-io" }
-#proxmox-lang = { path = "../proxmox/proxmox-lang" }
+proxmox-async = { path = "../proxmox/proxmox-async" }
+proxmox-auth-api = { path = "../proxmox/proxmox-auth-api" }
+proxmox-borrow = { path = "../proxmox/proxmox-borrow" }
+proxmox-compression = { path = "../proxmox/proxmox-compression" }
+proxmox-fuse = { path = "../proxmox-fuse" }
+proxmox-http = { path = "../proxmox/proxmox-http" }
+proxmox-io = { path = "../proxmox/proxmox-io" }
+proxmox-lang = { path = "../proxmox/proxmox-lang" }
 #proxmox-rest-server = { path = "../proxmox/proxmox-rest-server" }
-#proxmox-router = { path = "../proxmox/proxmox-router" }
-#proxmox-schema = { path = "../proxmox/proxmox-schema" }
-#proxmox-section-config = { path = "../proxmox/proxmox-section-config" }
-#proxmox-serde = { path = "../proxmox/proxmox-serde" }
-#proxmox-shared-memory = { path = "../proxmox/proxmox-shared-memory" }
+proxmox-router = { path = "../proxmox/proxmox-router" }
+proxmox-schema = { path = "../proxmox/proxmox-schema" }
+proxmox-section-config = { path = "../proxmox/proxmox-section-config" }
+proxmox-serde = { path = "../proxmox/proxmox-serde" }
+proxmox-shared-memory = { path = "../proxmox/proxmox-shared-memory" }
+proxmox-sortable-macro = { path = "../proxmox/proxmox-sortable-macro" }
 #proxmox-subscription = { path = "../proxmox/proxmox-subscription" }
-#proxmox-sys = { path = "../proxmox/proxmox-sys" }
-#proxmox-tfa = { path = "../proxmox/proxmox-tfa" }
-#proxmox-time = { path = "../proxmox/proxmox-time" }
-#proxmox-uuid = { path = "../proxmox/proxmox-uuid" }
-
+proxmox-sys = { path = "../proxmox/proxmox-sys" }
+proxmox-tfa = { path = "../proxmox/proxmox-tfa" }
+proxmox-time = { path = "../proxmox/proxmox-time" }
+proxmox-uuid = { path = "../proxmox/proxmox-uuid" }
 #proxmox-apt = { path = "../proxmox-apt" }
 #proxmox-openid = { path = "../proxmox-openid-rs" }
-
-#pxar = { path = "../pxar" }
+pxar = { path = "../pxar" }
 
 [features]
 default = []
 #valgrind = ["valgrind_request"]
+
+[package.metadata.generate-rpm]
+assets = [
+  { source = "target/release/pxar", dest = "/usr/bin/pxar", mode = "0755" },
+  { source = "target/release/proxmox-backup-client", dest = "/usr/bin/proxmox-backup-client", mode = "0755" }
+]
